runtime/cfdumper/main.c

```diff
@@ -998,6 +1008,35 @@ static void printClassFile(J9CfrClassFile* classfile)
 		}
 	}
 
+	/* JEP 360: sealed class permits list */
+	if(classfile->j9Flags & CFR_J9FLAG_IS_SEALED) {
+		/* find PermittedSubclasses attribute */
+		for (i = 0; i < classfile->attributesCount; i++) {
+			if (CFR_ATTRIBUTE_PermittedSubclasses == classfile->attributes[i]->tag) {
+				/* found attribute, print permitted subclasses */
+				j9tty_printf( PORTLIB, " permits ");
+
+				for (j = 0; j < ((J9CfrAttributePermittedSubclasses*)classfile->attributes[i])->numberOfClasses; j++) {
+					/* class index */
+					index = ((J9CfrAttributePermittedSubclasses*)classfile->attributes[i])->classes[j];
+					/* class name index */
+					index = classfile->constantPool[index].slot1;
+					string = classfile->constantPool[index].bytes;
+
+					k = 0;
+					while(string[k])
+					{
+						j9tty_printf( PORTLIB, "%c", (string[k] == '/')?'.':string[k]);
+						k++;
+					}
+					if (j + 1 != ((J9CfrAttributePermittedSubclasses*)classfile->attributes[i])->numberOfClasses) j9tty_printf( PORTLIB, ", ");
```

Comment:
Please bracket the addition, and use braces.

Reply:
I would like the brackets around the addition, please - the rest doesn't matter.

Reply:
updated.

Reply:
I'm not seeing the change.

Reply:
I think I misread the change you wanted. Should be there now.